### PR TITLE
Added HEALTHCHECK on the Dockerfile.armhf

### PR DESCRIPTION
**Pull request by sstassin**
I am using it since fews days, and it is working without any issue (see the screenshot, with one of my container is unhealthy).

It could also be interesting to explain that, according to the Docker Team's 
-  there is no plan to manage a restart, or stop of an unhealthy container.
- an unhealthy docker just run until it it stopped by a command
- this could be"easily done" by a bash command, that invoke a docker ps command to list all unhealthy containers and restart it (i could give you the code if you want)... scheduled in a cron job, you have a 24H/24H running container.

Regards and thank you for you great job.

Image 8

### CODE REVIEW

**Comment by sstassin** [comment]
I am sorry, i have once again included the intermédiates commit, and the change in the .md file.
Sorry for that, i will be more attentive next time

**Comment by sstassin** [comment]
Here is the screenshot of the unhealthy "transmission-dev" container,... that have just been restarted by my cronjob.

Image 9

If it can help someone....

**Comment by pkishino** [comment]
I think it would look cleaner if you squashed all commits into one :)
Apart from that it’s a good idea

File: README.md [hunk 1]
```diff
@@ -4,8 +4,11 @@
 [![Docker Pulls](https://img.shields.io/docker/pulls/haugene/transmission-openvpn.svg)](https://hub.docker.com/r/haugene/transmission-openvpn/)
 [![Join the chat at https://gitter.im/docker-transmission-openvpn/Lobby](https://badges.gitter.im/docker-transmission-openvpn/Lobby.svg)](https://gitter.im/docker-transmission-openvpn/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+
 ## Quick Start
 
+## WARNING : this fork is only a working fork... always refer to the orginal Haugene repository
```
Comment (pkishino) [inline README.md]: This should be removed if it is to be merged

Comment (sstassin) [reply README.md]: Sorry... i  am not very aware a about merging an excluding files.. i will do that asap. TY

**Comment by haugene** [comment]
Hey @sstassin and I'm sorry for the really late response here. Once again you find a good way to improve the image and I really appreciate it. I do agree with @pkishino though about squashing commits and removing the comment.

So I've just merged #1037 that also addresses this issue, and I'm closing this for now.
Please test it out as the health check should be up and running now. And let me know if there are still some issues.

**Pull request closed without merging**